easystacks/software.eessi.io/2023.06/eessi-2023.06-eb-4.8.2-2023a.yml

```diff
@@ -52,3 +52,7 @@ easyconfigs:
   - BWA-0.7.17-20220923-GCCcore-12.3.0.eb:
       options: 
         from-pr: 19820 
+  - gnuplot-5.4.8-GCCcore-12.3.0.eb: 
```

Comment:
This is already there by now, probably from another PR, so can be removed I think.